api/openapi/v1/core-metadata.yaml

```diff
@@ -171,75 +155,71 @@ paths:
           type: string
       responses:
         200:
-          description: list of addressable matching port
+          description: List of addressable matching port
           content:
             '*/*':
               schema:
                 $ref: '#/components/schemas/addressable'
         400:
-          description: for malformed or unparsable requests
+          description: For malformed or unparsable requests
         500:
-          description: for unknown or unanticipated issues
+          description: For unknown or unanticipated issues
   /v1/addressable/publisher/{publisher}:
     get:
-      description: Return Addressable objects with given publisher. List may be empty
-        if none are associated to the publisher. Returns Internal Service Error (HTTP
-        500) for unknown or unanticipated issues.
+      description: Return addressable objects with given publisher. List may be empty
+        if none are associated with the publisher.
       parameters:
       - name: publisher
         in: path
-        description: publisher name (as in MQTT publisher)
+        description: Publisher name
         required: true
         style: simple
         explode: false
         schema:
           type: string
       responses:
         200:
-          description: list of addressable matching publisher
+          description: List of addressable matching publisher
           content:
             '*/*':
               schema:
                 $ref: '#/components/schemas/addressable'
         400:
-          description: for malformed or unparsable requests
+          description: For malformed or unparsable requests
         500:
-          description: for unknown or unanticipated issues.
+          description: For unknown or unanticipated issues.
   /v1/addressable/topic/{topic}:
     get:
-      description: Return Addressable objects with given topic. List may be empty
-        if none are associated to the topic. Returns Internal Service Error (HTTP
-        500) for unknown or unanticipated issues.
+      description: Return addressable objects with given topic. List may be empty
+        if none are associated with the topic.
       parameters:
       - name: topic
         in: path
-        description: topic name (as in MQTT topic name)
+        description: Topic name, for example the MQTT topic name
```

Comment:
Remove the implementation detail. "Topic name, as used by the messaging provider"